src/_playground/_popper.scss

```diff
@@ -0,0 +1,128 @@
+$arrow-height: 8px;
```

Comment:
Thanks. 

we should be able to point fundamental-vue and furnamental-ngx to this same sass.

Reply:
Found ya. :)